### PR TITLE
Add extension FAQ (copied over from the repo) + translation + phishing info

### PR DESCRIPTION
**Pull request by Tbaut**

### CODE REVIEW

**Review by jacogr — APPROVED** [review]
!

File: docs/extension/FAQ.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+Add a file `translation.json` containing [the translation from the english](https://github.com/polkadot-js/extension/blob/master/packages/extension/public/locales/en/translation.json).
+Finally, add the new language in the selection menu by adding it to the array returned by the [getLanguageOptions](https://github.com/polkadot-js/extension/blob/master/packages/extension-ui/src/util/getLanguageOptions.ts#L12-L27).
+Once your PR is submitted and approved, the new language will be available in the next release of the extension.
+
+## I want to send funds directly from the extension.
+The extension is not meant to be a full wallet replacement. It tries to focus on pure account management while making this as smooth as possible. The extension simply makes the stored/imported accounts available to any dApp that can perform transfers, allow you to vote on democracy proposals and/or participate in any feature available on Polkadot and Substrate. We really aim to keep this singular account-only focus with no network connectivity, leaving the hard-work of understanding chains and their intricacies over to over to wallets such as [polkadot-js/apps](https://polkadot.js.org/apps/).
```
Comment (shawntabrizi) [inline docs/extension/FAQ.md:20]:
```suggestion
The extension is not meant to be a full wallet replacement. It tries to focus on pure account management while making this as smooth as possible. The extension simply makes the stored/imported accounts available to any dApp that can perform transfers, allows you to vote on democracy proposals, and participate in any features available on Polkadot and Substrate. We really aim to keep this singular account-only focus with no network connectivity, leaving the hard-work of understanding chains and their intricacies over to over to wallets such as [polkadot-js/apps](https://polkadot.js.org/apps/).
```

File: docs/extension/FAQ.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+## My addresses display differently on the extension vs a dapp.
```
Comment (shawntabrizi) [inline docs/extension/FAQ.md:22]: might be cool to have some way for a website to tell the extension to format the addresses differently while on a page.

would this be attackable/exploitable? (not really i would guess)

File: docs/extension/FAQ.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+Unlike a dapp, the extension itself does not connect to a network. The addresses are formatted with a network-specific prefix, which on a dapp, is retrieved upon connection from the chain properties. Underlying the public keys can be the same, but based on the address formatting the display could be different. This does not affect operation at all. You can choose which format to display the addresses in your extension via the "Options" screen.
```
Comment (shawntabrizi) [inline docs/extension/FAQ.md:23]:
```suggestion
Unlike a dapp, the extension itself does not connect to a network. The addresses are formatted with a network-specific prefix, which on a dapp, is retrieved upon connection from the chain properties. Under the surface, the public keys can be the same, but based on the address formatting the display could be different. This does not affect operation at all. You can choose which format to display the addresses in your extension via the "Options" screen.
```

Comment (shawntabrizi) [reply docs/extension/FAQ.md:23]: Another alternative fix is "The underlying public keys can be the same..."

File: docs/extension/FAQ.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+## What does it mean to derive an account?
+We can imagine that accounts are stored as a tree. It allows us to structure accounts by our own needs. The root account is created at first and all the new ones will be its children by default. If we want to derive from other accounts, it can be done by selecting Derive New Account option in the parent account’s context menu. Before deriving an account, the [HDKD derivation path](https://github.com/paritytech/parity-signer/wiki/HDKD-on-Parity-Signer#the-form-of-path) can be provided - it is a great tool that could be used for further account structuring.
+
+The format of a derivation path is `//<hard>/<soft>///<password>` where
+* `//hard` means that the connection between accounts is not “visible” for public addresses.
+* `/soft` connects accounts in a manner that can be proven.
+* `///password` provides an additional (optional) encryption.
+
+## Why can't I import an account with a hex-encoded private key?
```
Comment (shawntabrizi) [inline docs/extension/FAQ.md:33]: It makes sense that the extension will not allow the export of an account as hex private key, but why stop import? If I am a user with a hex private key, why not let me import it?

**Review by shawntabrizi — APPROVED** [review]
Really nice @Tbaut 

EDIT: But would be nicer if we split lines every 100-120 characters ;)

File: docs/extension/FAQ.md (hunk 4 above)
Comment (jacogr) [reply docs/extension/FAQ.md:33]: You can actually. However it is rather „hidden“ ie. It says „mnemonic“ but allows hex as well. 

So there are a couple of things -

- we want to drive to mnemonic only
- but don’t want to limit

So the naming and wording here may change (it is actually ancient text), but it is very unlikely that we will push anythibg but mnemonic.

Comment (shawntabrizi) [reply docs/extension/FAQ.md:33]: @jacogr good to hear, i think you position and the functionality you provide makes sense here.

Hidden feature is enough, as long as people aren't doing something crazy like forking the extension to hack in this feature.

Would say the content in this section is a LITTLE strong, but fine enough to me if you want to scare people away.

File: docs/extension/FAQ.md (hunk 3 above)
Comment (Tbaut) [reply docs/extension/FAQ.md:23]: Thanks, I think I prefer the second suggestion:
```suggestion
Unlike a dapp, the extension itself does not connect to a network. The addresses are formatted with a network-specific prefix, which on a dapp, is retrieved upon connection from the chain properties. The underlying public keys can be the same, but based on the address formatting, the display could be different. This does not affect operation at all. You can choose which format to display the addresses in your extension via the "Options" screen.
```

File: docs/extension/FAQ.md (hunk 2 above)
Comment (Tbaut) [reply docs/extension/FAQ.md:22]: This is supported already, you can pass a `ss58Format` option to the `web3Accounts` function, as specified here
https://github.com/polkadot-js/extension/blob/58b1320d4166e0703a680cb5cadaf9bffdfcf23f/packages/extension-dapp/src/index.ts#L100

Comment (shawntabrizi) [reply docs/extension/FAQ.md:33]: ah niceee

**Pull request merged**